### DISCUSSION TITLE
Not receive student benefits after get approved.

### DISCUSSION THREAD

**Discussion by ccyyhh53** [community/community#129715] [GitHub Education]
### Hi everyone,

Hi everyone,I registered for student developer pack and got passed for 4 days but dont get benefits.When can I get my benefits?Thank you.

**Comment by davevad93** [comment]
Hi @ccyyhh53 ,

Don't worry, it may take a few days or even a couple of weeks, as they are probably dealing with a high volume of requests.

When the benefits are available the status will change to:

`Congrats! Your request for @ccyyhh53 was approved and your benefits are now available`
